var/spack/repos/builtin/packages/neurodamus-neocortex/package.py

```diff
@@ -13,12 +12,17 @@ class NeurodamusNeocortex(NeurodamusModel):
     git      = "ssh://bbpcode.epfl.ch/sim/models/neocortex"
 
     version('develop', git=git, branch='master', submodules=True)
-    version('1.1', tag='1.1', submodules=True)
 
     variant('v5', default=True, description='Enable support for previous v5 circuits')
+    variant('plasticity',  default=False, description="Use optimized ProbAMPANMDA_EMS and ProbGABAAB_EMS")
+
+    mech_name = "neocortex"
 
     @run_before('merge_hoc_mod')
-    def include_v5(self):
+    def prepare_mods(self):
         if self.spec.satisfies('+v5'):
-            self.copy_all('mod/v5', 'mod', copyfunc=shutil.move)
-
+            copy_all('mod/v5', 'mod', copyfunc=copy_all.symlink2)
+        copy_all('mod/v6', 'mod', copyfunc=copy_all.symlink2)
```

Comment:
What is your thought about our discussion of building two specials in same module :  `v5/special` and `v6/special`  ?

Or, we will need to provide two modules : neurocortext/0.1/v5 and neurocortext/0.1/v6

Reply:
Oh, I thought we were building a single v5+v6 neurodamus, to behave like the previous neurodamus. But that's a idea I like, but then how so we setup PATH?

Reply:
I like the idea of a single module better. But if not possible, maybe?:

- neurodamus/neocortex-v5-0.1
- neurodamus/neocortex-v6-0.1